content/applications/sales/crm/track_leads/marketing_attribution.rst

```diff
@@ -0,0 +1,142 @@
+
+Odoo can use these UTMs as parameters in the marketing attribution report to track the metrics and
+performance of marketing campaigns.
+
+Create UTMs
+-----------
+
+The :doc:`link tracker <../../../websites/website/reporting/link_tracker>` in Odoo can be used to
+create and configure UTMs.
+
+UTMs can also be automatically generated by :doc:`Email Marketing
+<../../../marketing/email_marketing>` and :doc:`Marketing Automation
+<../../../marketing/marketing_automation>` apps.
+
+The UTM parameters used in a marketing attribution report are *Medium*, *Source*, and *Campaign* in
+descending order of coverage.
+
+- *Medium* is the UTM with the widest coverage, such as social media.
+- *Source* is more narrow and covers parameters such as the specific social media platform.
+- *Campaign* is the narrowest and can track specific marketing campaigns.
+
+Create reports
+==============
+
+To start creating a report, click the :icon:`fa-caret-down` :guilabel:`(down caret)` to the right of
+the search bar to see the list of filtering and grouping parameters.
+
+:guilabel:`Filters`, located in the left column of the search options, can be used to keep only
+the results that fit the filter. For example, selecting the :guilabel:`Won` filter will only
+show won leads in the attribution report. :guilabel:`Group By`, found in the middle column, is
+used to organize the results into groups and can be used with or without filters.
+
+.. image:: marketing_attribution/search-results-multiple-options.png
+   :align: center
+   :alt: Select any number of filters and groups in the search options.
+
+.. tip::
+    Setting multiple Group By options will create nested groups according to which is selected
+    first. For example, selecting :guilabel:`Medium` followed by :guilabel:`Source` and then
+    ::guilabel:`Campaign` in :guilabel:`Group By` will sort all results *first* by medium, *then* by
+    the specific sources in each medium, followed by the campaigns in each source.
+
+    This can be verified by looking at the direction of the carat in the group tile in
+    the search bar.
+
+    .. image:: marketing_attribution/group-by.png
+       :align: center
+       :alt: The text in the tile is `Country > City`, showing that city is a subgroup of country.
+
+.. example::
+    For a useful first report:
+
+    #. Select the :guilabel:`Active` filter to view only leads that are still marked as active.
+    #. Select :guilabel:`Source` followed by the :guilabel:`City` or :guilabel:`Country`
+       groups depending on which grouping is more relevant.
+
+    .. image:: marketing_attribution/campaign-and-country-groups.png
+       :align: center
+       :alt: Each lead is now sorted by source, followed by city or country.
+
+    This report will contain all active leads grouped first by the source of the lead, then the
+    country or city each lead is from. This is useful to see the density of active opportunities
```

Comment:
```suggestion
    city or country each lead is from. This is useful to see the density of active opportunities
```
suggesting this minor reordering to fall in line with the sentence formatting above